### ISSUE TITLE
Context menu is always in English

### ISSUE THREAD

**Issue by yileifeng** [ramp4-pcar4/highcharts-editor#115]
## Description

From @james-rae in demo derby, context menu of highcharts preview is always in English: 

<img width="748" height="397" alt="Image" src="https://github.com/user-attachments/assets/cd2c9fd7-5234-42b4-b891-eb348bd032b3" />

This should be set based on the app language or if not defined, use the i18n locale.
